Formula/thrift@0.9.rb

```diff
@@ -0,0 +1,63 @@
+class ThriftAT09 < Formula
+  desc "Framework for scalable cross-language services development"
+  homepage "https://thrift.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=/thrift/0.9.3/thrift-0.9.3.tar.gz"
+  sha256 "b0740a070ac09adde04d43e852ce4c320564a292f26521c46b78e0641564969e"
+
+  keg_only :versioned_formula
+
+  option "with-haskell", "Install Haskell binding"
+  option "with-erlang", "Install Erlang binding"
+  option "with-java", "Install Java binding"
+  option "with-perl", "Install Perl binding"
+  option "with-php", "Install PHP binding"
+  option "with-libevent", "Install nonblocking server libraries"
+
+  depends_on "bison" => :build
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "libevent" => :optional
+  depends_on :python => :optional
+
+  def install
+    system "./bootstrap.sh" unless build.stable?
+
+    exclusions = ["--without-ruby", "--disable-tests", "--without-php_extension"]
+
+    exclusions << "--without-python" if build.without? "python"
+    exclusions << "--without-haskell" if build.without? "haskell"
+    exclusions << "--without-java" if build.without? "java"
+    exclusions << "--without-perl" if build.without? "perl"
+    exclusions << "--without-php" if build.without? "php"
+    exclusions << "--without-erlang" if build.without? "erlang"
+
+    ENV.cxx11 if MacOS.version >= :mavericks && ENV.compiler == :clang
+
+    # Don't install extensions to /usr:
+    ENV["PY_PREFIX"] = prefix
+    ENV["PHP_PREFIX"] = prefix
+
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}",
+                          "--libdir=#{lib}",
+                          "--with-openssl=#{Formula["openssl"].opt_prefix}",
+                          *exclusions
+    ENV.deparallelize
+    system "make"
+    system "make", "install"
+  end
+
+  def caveats; <<-EOS.undent
+    To install Ruby binding:
+      gem install thrift
+
+    To install PHP extension for e.g. PHP 5.5:
+      brew install homebrew/php/php55-thrift
+  EOS
+  end
+
+  test do
+    assert_match /Thrift/, shell_output("#{bin}/thrift --version", 0)
+    assert_match /0\.9\.3/, shell_output("#{bin}/thrift --version", 0)
```

Comment:
Why not combine this to `assert_match "Thrift version #{version}", shell_output("#{bin}/thrift --version", 0)`?